dts/arm/nordic/nrf5340_cpuappns.dtsi

```diff
@@ -29,7 +29,7 @@
 	};
 
 	chosen {
-		zephyr,entropy = &cryptocell_sw;
+		zephyr,entropy = &psa_rng;
```

Comment:
I think this will break in the next upmerge, the fromlist will be reverted, at which point `zephyr,entropy = &cryptocell_sw;` will be back again, then https://github.com/zephyrproject-rtos/zephyr/pull/53820/files will be applied, which is going to look for `zephyr,entropy = &rng_hci;` which does not exist

Reply:
> I think this will break in the next upmerge, the fromlist will be reverted, at which point `zephyr,entropy = &cryptocell_sw;` will be back again, then https://github.com/zephyrproject-rtos/zephyr/pull/53820/files will be applied, which is going to look for `zephyr,entropy = &rng_hci;` which does not exist

Yeah, the `[nrf noup] dts: choose cryptocell for entropy when available` patch needs to be rewritten.

I have done that now, so that this will not break in the upmerge. It could have been done then as well. It does not really matter.